plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1278,8 +1279,14 @@ blockly.block.feature_huge_brown_mushroom=Huge brown mushroom feature with cap:
 blockly.block.feature_huge_red_mushroom=Huge red mushroom feature with cap: %1 stem: %2 radius: %3
 blockly.block.feature_huge_fungus=Huge fungus feature with stem: %2 hat: %3 decor: %4 planted: %5, only on ground: %1
 blockly.block.feature_lake=Lake feature with fluid: %1 border: %2
+blockly.block.feature_ore=Ore feature with size: %1 %2discard if exposed to air with chance: %3
+blockly.block.feature_ore.empty=(Empty list)
 blockly.block.feature_random_patch_simple=Patch feature with block: %4 condition: %5 number of tries: %1 XZ spread: %2 Y spread: %3
+blockly.block.feature_replace_single_block=Single block ore feature
+blockly.block.feature_replace_single_block.empty=(Empty list)
 blockly.block.feature_replace_sphere=Blob feature replacing: %1 with: %2 radius: %3
+blockly.block.feature_scattered_ore=Scattered ore feature with size: %1 %2discard if exposed to air with chance: %3
```

Comment:
There could probably be space, although it does not matter visually it is good to have separation for translators: `%2discard`